amethyst_tiles/shaders/tiles.vert

```diff
@@ -0,0 +1,38 @@
+#version 450
+
+layout(std140, set = 0, binding = 0) uniform ViewArgs {
+    uniform mat4 proj;
+    uniform mat4 view;
+};
+
+// Quad transform.
+layout(location = 0) in vec2 dir_x;
+layout(location = 1) in vec2 dir_y;
+layout(location = 2) in vec2 pos;
+layout(location = 3) in vec2 u_offset;
+layout(location = 4) in vec2 v_offset;
+layout(location = 5) in float depth;
```

Comment:
No, I want them to be different shaders because although they are similar for now, in the future I see this becoming more optimized and specialized to map use cases so i want them to be different to begin with.